llvm/lib/Target/AArch64/AArch64Features.td

```diff
@@ -472,22 +472,22 @@ def FeatureFP8 : ExtensionWithMArch<"fp8", "FP8", "FEAT_FP8",
   "Enable FP8 instructions", [FeatureFAMINMAX, FeatureLUT, FeatureBF16]>;
   
 def FeatureFP8FMA : ExtensionWithMArch<"fp8fma", "FP8FMA", "FEAT_FP8FMA",
-  "Enable fp8 multiply-add instructions", [FeatureFP8]>;
+  "Enable FP8 multiply-add instructions", [FeatureFP8]>;
```

Comment:
No architecture for these ones?

Reply:
I've not added Arm architecture versions where they didn't previously exist. In this case, I'll add it (Armv9.5-A)